.rubocop_todo.yml

```diff
@@ -1,3 +1,7 @@
+Style/BlockDelimiters:
+  Exclude:
+    - 'spec/controllers/api/actions_controller_spec.rb'
```

Comment:
Since using curly braces for `expect` blocks is pretty much a standard, what do you think about disabling this for `spec/*`?